File: webservices/CORE/Provincia.php
<?php


class Provincia
{
    public $idprovincia,$nombre,$estado;

    /**
     * Provincia constructor.
     * @param $idprovincia
     * @param $nombre
     * @param $estado
     */
    public function __construct($idprovincia, $nombre, $estado)
    {
        $this->idprovincia = $idprovincia;
        $this->nombre = $nombre;
        $this->estado = $estado;
    }

    /**
     * @return mixed
     */
    public function getIdprovincia()
    {
        return $this->idprovincia;
    }

    /**
     * @param mixed $idprovincia
     */
    public function setIdprovincia($idprovincia)
    {
        $this->idprovincia = $idprovincia;
    }

    /**
     * @return mixed
     */
    public function getNombre()
    {
        return $this->nombre;
    }

    /**
     * @param mixed $nombre
     */
    public function setNombre($nombre)
    {
        $this->nombre = $nombre;
    }

    /**
     * @return mixed
     */
    public function getEstado()
    {
        return $this->estado;
    }

    /**
     * @param mixed $estado
     */
    public function setEstado($estado)
    {
        $this->estado = $estado;
    }


}

File: Facturador/app/src/main/java/com/example/facturador/Activity/Model/Provincia.java
package com.example.facturador.Activity.Model;

public class Provincia {

    private int idprovincia;
    private String nombre;
    private  String estado;
    private int id_base;

    public Provincia() {

    }

    public int getId_base() {
        return id_base;
    }

    public void setId_base(int id_base) {
        this.id_base = id_base;
    }

    public Provincia(int idprovincia, String nombre, String estado) {
        this.idprovincia = idprovincia;
        this.nombre = nombre;
        this.estado = estado;
    }

    public int getIdprovincia() {
        return idprovincia;
    }

    public void setIdprovincia(int idprovincia) {
        this.idprovincia = idprovincia;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getEstado() {
        return estado;
    }

    public void setEstado(String estado) {
        this.estado = estado;
    }
}
